repo2docker/buildpacks/r.py

```diff
@@ -97,6 +99,9 @@ def r_version(self):
                 # available. Users can however explicitly specify the full version to get something specific
                 if r_version in version_map:
                     r_version = version_map[r_version]
+                elif len(r_version.split(".")) == 1:
```

Comment:
```suggestion
                elif len(r_version.split(".")) == 2:
```
Shouldn't this be 2?